ocs_ci/ocs/node.py

```diff
@@ -723,3 +723,67 @@ def check_nodes_specs(min_memory, min_cpu):
         f"{min_cpu} CPUs and {min_memory} Memory"
     )
     return True
+
+
+def delete_and_create_osd_node_aws_ipi(osd_node_name):
+    """
+    Unschedule, drain and delete osd node, and creating a new osd node.
+    At the end of the function there should be the same number of osd nodes as
+    it was in the beginning, and also ceph health should be OK.
+    This function is for AWS IPI.
```

Comment:
Is it really AWS IPI specific? This **should** work in the same way in any IPI platform. If that is not the case, it's a bug.

Reply:
Currently, we still have this bug https://bugzilla.redhat.com/show_bug.cgi?id=1844521. 
So for vSphere, we need to wait until it will be fixed.

Reply:
but the question is if this function will serve vSphere as well. If so, let's remove `aws` from its name

Reply:
@yitzhak12 please check this comment

Reply:
Originally this has been made by @prsurve and he implemented it for AWS IPI - I just moved it to **ocs_ci/ocs/node.py**. So I guess it will be better to ask him for his opinion.